content/en/blog/_posts/2024-08-13-Kubernetes-v1-31-Release.md

```diff
@@ -0,0 +1,432 @@
+
+## Highlights of features graduating to Beta
+
+_This is a selection of some of the improvements that are now beta following the v1.31 release._
+    
+### nftables backend for kube-proxy
+
+The Kubernetes project intends to update kube-proxy on Linux so that it uses `nftables`,
+not `iptables`, as the default implementation. The nftables backend moves to beta in v1.31,
+behind a newly added feature gate, `NFTablesProxyMode` (disabled by default).
+In this mode, kube-proxy configures packet forwarding rules using the nftables API of the kernel netfilter subsystem. 
+For each endpoint, it installs nftables rules which, by default, select a backend Pod at random.
+
+The nftables API is the successor to the iptables API and is designed to provide better performance and scalability than iptables. 
+The `nftables` proxy mode is able to process changes to service endpoints faster and more efficiently than the `iptables` mode, and is also able to more efficiently process packets in the kernel (though this only
+becomes noticeable in clusters with tens of thousands of services).
+
+As of Kubernetes v1.31, the `nftables` mode is still relatively new, and may not be compatible with all network plugins; consult the documentation for your network plugin. 
+This proxy mode is only available on Linux nodes, and requires kernel 5.13 or later.
+See the full [migration guide](/docs/reference/networking/virtual-ips/#proxy-mode-nftables) for more details. 
+
+This work was done as part of [KEP #3866](https://github.com/kubernetes/enhancements/issues/3866) by [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+
+### Changes to reclaim policy for PersistentVolumes
+
+The Always Honor PersistentVolume Reclaim Policy feature has advanced to beta in Kubernetes v1.31. 
+This enhancement ensures that the PersistentVolume (PV) reclaim policy is respected even after the associated PersistentVolumeClaim (PVC) is deleted, thereby preventing the leakage of volumes.
+
+Prior to this feature, the reclaim policy linked to a PV could be disregarded under specific conditions, depending on whether the PV or PVC was deleted first. 
+Consequently, the corresponding storage resource in the external infrastructure might not be removed, even if the reclaim policy was set to "Delete". 
+This led to potential inconsistencies and resource leaks.
+
+With the introduction of this feature, Kubernetes now guarantees that the "Delete" reclaim policy will be enforced, ensuring the deletion of the underlying storage object from the backend infrastructure, regardless of the deletion sequence of the PV and PVC.
+
+This work was done as a part of [KEP #2644](https://github.com/kubernetes/enhancements/issues/2644) and by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+    
+### Bound service account token improvements
```

Comment:
moving conversation here, https://github.com/kubernetes/website/pull/47281#discussion_r1703975446

@ritazh and @enj I've tried to update the content here based on the comments in the [slack thread](https://kubernetes.slack.com/archives/C0EN96KUY/p1722015619684019). Can you review? thanks!

Reply:
@liggitt do you have time to review this section?

Reply:
@liggitt thank you for the review. I've updated this section based on your comments.

Reply:
this section lgtm (aside from the nits @katcosgrove already mentioned)